modules/common/test/helpers/volumes.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2023 Red Hat
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+// AssertVolumeExists - asserts the existence of a named volume in a []corev1.Volume.
+//
+// Example usage:
+//
+//	th.AssertVolumeExists("name", []corev1.Volume{...})
+func (tc *TestHelper) AssertVolumeExists(name string, volumes []corev1.Volume) bool {
+	exist := false
+	for _, v := range volumes {
+		if v.Name == name {
+			exist = true
+		}
+	}
+	gomega.Expect(exist).To(gomega.BeTrue())
```

Comment:
I think this can be "simplified" (make it more gomegalike):
```golang
gomega.Expect(volumes).ContainElement(MatchField("Name", name))
```

Reply:
cool , thanks @gibizer will push an update

Reply:
thanks